commons/src/main/java/io/hyscale/commons/component/IInterceptorProcessor.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2019 Pramati Prism, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.hyscale.commons.component;
+
+import io.hyscale.commons.exception.HyscaleException;
+
+public interface IInterceptorProcessor {
+    
+    public void around(Object... args) throws HyscaleException;
```

Comment:
Do we need around advice here ???
The naming of each method should be in Interceptor terminology rather than aspect terminology like 
before ----> preProcess
after ------> postProcess